examples/platforms/nrf52840/alarm.c

```diff
@@ -67,6 +68,10 @@
 #define US_PER_OVERFLOW     (512UL * US_PER_S)  ///< Time that has passed between overflow events. On full RTC speed, it occurs every 512 s.
 
 #define MS_PER_S            1000UL
+
+#if OPENTHREAD_CONFIG_ENABLE_TIME_SYNC
+#define XTAL_ACCURACY       40 // The crystal used on nRF52840PDK has ±20ppm accuracy.
```

Comment:
Avoid wrapping these kinds of definitions to ensure that there are no symbol redefinitions even when `OPENTHREAD_CONFIG_ENABLE_TIME_SYNC` is 0.